package Exchange;

message StringParameter
{
  required string Data=1;
}